### ISSUE TITLE
Feature Request: Group notifications that belong to the same post

### ISSUE THREAD

**Issue by fbanning** [LucasGGamerM/moshidon#222]
**Is your feature request related to a problem? Please describe.**
Currently the notification tab looks pretty busy and every fav/boost takes up a lot of space.

**Describe the solution you'd like**
Group notifications that belong to the same post.

**Describe alternatives you've considered**
For example phanpy.social does this in a very nice way in the notification section and maybe similar behaviour can be achieved in moshidon? In terms of native Android apps there's fedilab which also groups repeated notifications (favs and boosts) for the same post and reduces visual clutter by that.

**Additional context**
Currently nothing. I could provide some screenshots from phanpy.social and fedilab, if that's helpful in any way.

**Comment by CelianGdfrd** [comment]
Agreed. Notifications become unreadable just after 10, 15 likes on one post.

**Comment by micr0-dev** [comment]
it would be so awsum to have this 🙏 

its already in mastodon-android upstream so it just needs to be merged in
